dotCMS/src/main/java/com/dotcms/util/pagination/ContainerPaginator.java

```diff
@@ -0,0 +1,65 @@
+package com.dotcms.util.pagination;
+
+import com.dotcms.contenttype.model.type.ContentType;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotcms.contenttype.model.type.ContentType'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)